templates/common/footer.tpl

```diff
@@ -22,7 +22,7 @@
 	</div>
 </div>
 
-<a href="#" class="requestHelpPanel" data-topic="hi">
+<a href="#" class="requestHelpPanel" data-topic="chapter_6_submissions.md">
```

Comment:
I like the cleanup in general, but obviously we'd need to place the help links in the right markup; do you mind dropping one or two examples in as part of this PR, and removing the link from here?